lms/templates/courseware/dates.html

```diff
@@ -78,6 +78,9 @@ <h2 class="hd hd-2 date-title">
                                         % else:
                                             ${block.title}
                                         % endif
+                                        % if extra_info:
+                                            <span></span>
```

Comment:
Should this be adding in the extra info? Like `<span>${extra_info}</span>` or something similar